src/app/cdk/panel/panels/panels.component.html

```diff
@@ -43,11 +43,28 @@ <h3 class="activity-header orc-font-body-large">
               *ngIf="
                 (!currentAmount && total == 0) || (!currentAmount && !total)
               "
-              >(0)</span
+            >(0)</span
             >
             <span *ngIf="!currentAmount && currentAmount !== 0"></span>
-          </h3>
-        </div>
+          </ng-container>
+        </h3>
+        <ng-container *ngIf="type === 'peer-review'">
+              <span>
+                ({{ reviewsNumber }}
+                <ng-container *ngIf="reviewsNumber> 1" i18n="@@peerReview.reviews">
+                  reviews
+                </ng-container>
+                <ng-container *ngIf="reviewsNumber === 1" i18n="@@peerReview.reviews">
+                  review
+                </ng-container>
+                <ng-container i18n="@@peerReview.for">for</ng-container>
+                {{ currentAmount }}
+                <ng-container *ngIf="currentAmount > 1" i18n="@@peerReview.publicationsGrants"
+                >publications/grants</ng-container>
+                <ng-container *ngIf="currentAmount === 1" i18n="@@peerReview.publicationsGrants"
```

Comment:
@DanielPalafox here you are using the same key on both cases, shouldnt be a different case for singular than the one we use for plural? 

Looking at other parts of your PR, i think you use `peerReview.publicationsGrants` for plural and `peerReview.publicationGrant` for singular, right?